README.md

```diff
@@ -1,19 +1,42 @@
 # Info
+
 The AirConsole-Unity-Plugin is a C# wrapper for the AirConsole Javascript API.
-AirConsole provides a simple to use Javascript API for game developers to build 
-their own local multiplayer realtime browser games.
-You can find the Javascript API documentation here: http://developers.airconsole.com/#/api
+AirConsole provides a simple to use Javascript API for game developers to build their own local multiplayer realtime browser games.
+You can find the Javascript API documentation here: <http://developers.airconsole.com/#/api>
 
-IMPORTANT: The plugin comes with an embedded webserver / websocket-server for the communication between the AirConsole backend and the Unity-Editor. 
+IMPORTANT: The plugin comes with an embedded webserver / websocket-server for the communication between the AirConsole backend and the Unity-Editor.
 You don't need to install any other webserver or services.
 
+# Upgrade Notes
+
+## Upgrading from v2.11 to v2.12+
+
+The location of the Webview has changed in this release.
+
+When upgrading from v2.11 and before, if the Unity package import does not detect the move, please manually remove the old Webview plugin parts:
```

Comment:
Note: I think using "please" and other polite words is not necessary, being direct is better in general for documentation.